### PR TITLE
Adding version as param in script

### PR DESCRIPTION
**Pull request by anshuman-goel**
## Summary of the Pull Request

_What is this about?_
I need to modify `__version__.py` for every change and then build using `build_cli.ps1`. Introducing this change I can invoke it like `build_cli.ps1 -version "<version_number>`

## PR Checklist
* [ ] Applies to work item: #xxx
* [x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request
https://www.cyberciti.biz/faq/bash-shell-find-out-if-a-variable-has-null-value-or-not/

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_
Can be tested by running `.\build_cli.ps1 -version "1.3.1-78b655501368ce5e78bda632834f2e5ecdfd1bbb"` and `.\build_cli.ps1`

### CODE REVIEW

File: src/ci/unset-versions.sh [hunk 1]
```diff
@@ -7,7 +7,12 @@ set -ex
 
 SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
 GET_VERSION=${SCRIPT_DIR}/get-version.sh
```
Comment (anshuman-goel) [inline src/ci/unset-versions.sh:9]: it looks like we don't need `GET_VERSION` and `VERSION` during unset. will remove it.

Comment (anshuman-goel) [reply src/ci/unset-versions.sh]: removed

File: src/ci/set-versions.sh [hunk 2]
```diff
@@ -7,7 +7,12 @@ set -ex
 
 SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
 GET_VERSION=${SCRIPT_DIR}/get-version.sh
-VERSION=$(${GET_VERSION})
```
Comment (bmc-msft) [inline src/ci/set-versions.sh:10]: Please use:

```bash
VERSION=${1:-$(${GET_VERSION})}
```

File: src/ci/unset-versions.sh [hunk 3]
```diff
@@ -6,8 +6,6 @@
 set -ex
 
 SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
-GET_VERSION=${SCRIPT_DIR}/get-version.sh
```
Comment (bmc-msft) [inline src/ci/unset-versions.sh:9]: 👍

**Review by bmc-msft — CHANGES_REQUESTED** [review]

**Review by bmc-msft — APPROVED** [review]

**Comment by bmc-msft** [comment]
Testing in progress.

**Pull request merged**